mozilla-release/toolkit/mozapps/extensions/internal/XPIProvider.jsm

```diff
@@ -7953,6 +7957,7 @@ Object.assign(SystemAddonInstallLocation.prototype, {
       logger.warn(`System add-on ${aAddon.id} isn't restartless.`);
       return false;
     }
+#endif
```

Comment:
Did you already try simulating an update for this extension?
I think it may be somehow affected by these changes.